flow-server/src/test/java/com/vaadin/flow/server/frontend/FrontendUtilsTest.java

```
@@ -50,12 +50,24 @@
 import static com.vaadin.flow.server.Constants.VAADIN_SERVLET_RESOURCES;
 import static com.vaadin.flow.server.InitParameters.SERVLET_PARAMETER_STATISTICS_JSON;
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
 public class FrontendUtilsTest {
 
     private static final String USER_HOME = "user.home";
 
+    private static Class<?> CACHE_KEY;
+
+    static {
+        try {
+            CACHE_KEY = Class.forName("com.vaadin.flow.server.frontend.FrontendUtils$Stats");
+        } catch (ClassNotFoundException e) {
+            System.err.println("Could not access cache key for stats.json!");
```

Please replace `System.err.println` with `Assert.fail`.
This would make it fail fast and easier to investigate the failed tests in `FrontendUtilsTest`.

---

Done :)